com.unity.ml-agents/Runtime/Sensor/ISensor.cs

```diff
@@ -14,28 +24,28 @@ public interface ISensor
     {
         /// <summary>
         /// Returns the size of the observations that will be generated.
-        /// For example, a sensor that observes the velocity of a rigid body (in 3D) would return new {3}.
-        /// A sensor that returns an RGB image would return new [] {Width, Height, 3}
+        /// For example, a sensor that observes the velocity of a rigid body (in 3D) would return
+        /// new {3}. A sensor that returns an RGB image would return new [] {Width, Height, 3}
```

Comment:
```suggestion
        /// new {3}. A sensor that returns an RGB image would return new [] {Height, Width, 3}
```
oops

Reply:
Good catch. Did a global search and found another similar typo in Observation.cs (made the fix)